src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml

```diff
@@ -37,6 +37,13 @@
                 </th>
             </tr>
         </thead>
+        <tbody class="govuk-table__body">
+            <tr>
+                <td colspan="2">
+                    <p>There are currently no PCS obligations uploaded</p>
```

Comment:
there is this no results partial -"~/Areas/Aatf/Views/ManageEvidenceNotes/Overview/_NoResultsPartial.cshtml. Have a look if you can use it. you can simply pass the display message within.

Reply:
As meg says see if can use the no results partial...

Reply:
That requires a model of type ManageManageEvidenceNoteOverviewViewModel, could create a specific one for ManageObligations?

Reply:
@JohnPadgettCivica ?

Reply:
Thats fine we can leave it if we can't re-use...